Code/GraphMol/Canon.cpp

```diff
@@ -908,8 +906,8 @@ void canonicalizeFragment(ROMol &mol, int atomIdx,
   INT_VECT cyclesAvailable(MAX_CYCLES, 1);
 
   VECT_INT_VECT cycles(nAtoms);
-  for (VECT_INT_VECT_I vviIt = cycles.begin(); vviIt != cycles.end(); ++vviIt)
-    vviIt->resize(0);
+  for (auto & cycle : cycles)
```

Comment:
A simple matter of taste probably because I'm not as used to reading 'auto' in code yet. I saw this read it as the 'bitwise-and of auto and cycle'. If there was no space between the ampersand the 'cycle' variable, don't think my brain would have been tricked in that way. 

```
for (auto &cycle : cycles)
```

I think that's more readable, don't know if clang-tidy will let you make that subtle change.